addOns/paramminer/src/main/javahelp/org/zaproxy/addon/paramminer/resources/help_ar_SA/contents/about.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
```

Comment:
Only the main (English) files should be added.

Reply:
Done.. Did the same for messages